cmd/scollector/collectors/dotnet_windows.go

```diff
@@ -60,7 +60,7 @@ func c_dotnet_loading() (opentsdb.MultiDataPoint, error) {
 	}
 	var md opentsdb.MultiDataPoint
 	for _, v := range dst {
-		if !nameMatches(v.Name, regexesDotNet) {
+		if !util.NameMatches(v.Name, regexesDotNet) {
```

Comment:
This file is missing a "bosun.org/util" import. Adding that should fix travis ci builds